src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+       action="/search"
+       method="get"
+       data-qa="big"
+       class="form"
+       >
+      <label
+         for="big icon">
+        <img
+           src="./images/Search.svg"
+           class="icon--cursor icon-big"
+           alt="big Icon"
+   >
+      </label>
+
+      <input
+         type="text"
+         class="input-big input-big--appearance"
```

Comment:
class for bigger input can be just `input`, you will add modifiers for smaller one to be `input input--small`, don't need also that `input-big--appearance`, please move `--appearance` class styling to input to be using for both inputs looks the same :) 

```suggestion
         class="input"
```